### ISSUE TITLE
Re-implement rp9 support

### ISSUE THREAD

**Issue by giantclambake** [BlitterStudio/amiberry#1483]
With the new directory structure, we have the ~/Amiberry/rp9/ directory. If one copies a .rp9 files into this directory, it shows up in the Configurations panel ...however, the file does not load, and the emulation starts with the basic A500 config.

I understand this support was removed some time ago, so it really should be reinstated at some time, to properly support the functionality already evident.

TIA

**Comment by midwan** [comment]
Only the path has remained there, and I'm not really sure it's worth the trouble implementing this again.
I was looking at the specs and did some tests with WinUAE that supports it, and it seems to me that it's not very useful for us.
WHDLoad support is much more convenient, when possible.

RP9 supports multiple disk images as well, but I couldn't find a way to swap disks when loading one of those. They are all contained in the RP9 archive itself as ADFs, but they are not available to insert in the drive after the title has started, nor in the DiskSwapper for instance (at least on WinUAE).

I think it's significant work for very little benefit, considering the adoption it has out there.

**Comment by midwan** [comment]
considering the above, I think it's best if we just "hide" the RP9 folder (leave the config in place, but don't auto-create it during installation)

**Comment by giantclambake** [comment]
Yeah, seeing as that's the case, the best option would be to hide things, so it doesn't _look like_ it's possible ;)
